setup.py

```diff
@@ -220,6 +220,7 @@ def get_krb5_lib_path(
         "context",
         ("context_mit", "krb5_init_secure_context"),
         "creds",
+        ("change_password", "krb5_set_password"),
```

Comment:
Is there a reason why we do the check, seems like it's present on both MIT and Heimdal (and macOS Heimdal). From what I've seen the `krb5_set_password*` APIs can be unconditionally compiled/imported but `krb5_change_password` will need a conditional check as it's not present on macOS' Heimdal.

Reply:
sorry for the mess, see [below](https://github.com/jborean93/pykrb5/pull/50#discussion_r1730910751)